recipe/meta.yaml

```diff
@@ -16,13 +16,14 @@ build:
 
 requirements:
   host:
-    - python >=3.7,<3.10
+    - python >=3.10,<3.13
     - pip
   run:
-    - python >=3.7,<3.10
+    - python >=3.10,<3.13
     - setuptools
-    - openff-toolkit >=0.10.6, <0.11.0
-    - smirnoff-plugins >=0.0.3
+    - openff-toolkit >=0.11
+    - openff-interchange-base >=0.3.25
+    - smirnoff-plugins >=2023
```

Comment:
This (2023.08.0) seems to include the most recent relevant change (periodic/non-periodic vdW methods)